etc/uigui_uncrustify.ini

```diff
@@ -2831,7 +2831,7 @@ ValueDefault=0
 
 [Indent Macro Brace]
 Category=2
-Description="<html>How to indent within a macro followed by a brace on the same line<br/>This allows reducing the indent in macros that have (for example)<br/>`do { ... } while (0)` blocks bracketing them.<br/><br/>true:  add an indent for the brace on the same line as the macro<br/>false: do not add an indent for the brace on the same line as the macro<br/><br/>Default: true</html>"
+Description="<html>How to indent within a macro followed by a brace on the same line<br/>This allows reducing the indent in macros that have (for example)<br/>`do { ... } while ` blocks bracketing them.<br/><br/>true:  add an indent for the brace on the same line as the macro<br/>false: do not add an indent for the brace on the same line as the macro<br/><br/>Default: true</html>"
```

Comment:
```do { ... } while (0)``` should remain (the while condition is part of the do-while construct). I know that the output for the autogenereated cli test (see may other comment on PR conversation) is missing this ```(0)```: I think we should fix the autogeneration to make sure that ```(0)``` is there as well. This can be done separately from this PR.

Reply:
Done